.cirrus.yml

```diff
@@ -226,14 +229,40 @@ report_stageone_task:
         pip3 install avocado-framework==94.0 ${PIP_EXTRA}
     # run the unittests separately as they require a different PYTHONPATH in
     # order for the imports to work properly under avocado
-    unittest_script: PYTHONPATH=. avocado run tests/unittests/
-    main_script: PYTHONPATH=tests/ avocado run -p TESTLOCAL=true --test-runner=runner -t stageone tests/{cleaner,collect,report,vendor}_tests
+    unittest_script: &unit_test |
+        PYTHONPATH=. avocado run tests/unittests/
+    main_script: &stageone_test
+        PYTHONPATH=tests/ avocado run -p TESTLOCAL=true --test-runner=runner -t stageone tests/{cleaner,collect,report,vendor}_tests
     on_failure:
         fail_script: &faillogs |
             ls -d /var/tmp/avocado* /root/avocado* 2> /dev/null | xargs tar cf sos-fail-logs.tar
         log_artifacts: &logs
             path: "sos-fail-logs.tar"
 
+report_stageone_daily_task:
+    alias: "stageone_daily_report"
+    name: "Report Stage One - ${UBUNTU_DEVEL_FAMILY_NAME}"
+    allow_failures: true
+    depends_on:
+        - snap_build
+    gce_instance:
+        image_project: ${UBUNTU_DEVEL_PROJECT}
+        image_family: ${UBUNTU_DEVEL_FAMILY_NAME}
+        type: e2-medium
+    environment:
+      PKG: "snap"
+    setup_script: *setup
+    avocado_install_script: &avocado_setup |
+        pip3 uninstall -y avocado-framework --break-system-packages
+        git clone --single-branch -b 92lts https://github.com/avocado-framework/avocado.git
+        cd avocado
+        make install
```

Comment:
doing this to ensure to get the latest from the `92lts` branch so that it has the support for python3.12. Once they publish to pypi, we can then grab the 92.X lts.

thoughts?

Reply:
Seems good as a workaround for the time being until we wait for the new 92.X LTS is released into pypi.

Reply:
Maybe worth opening an sos issue to keep track this as a kind of TODO point?

Reply:
I agree, this will be tracked in #3588, That issue will not close till we have a permanent good solution